### PR TITLE
[docs] Document FROM_JSON transformer

### LINKED ISSUE #3137: Document `FROM_JSON` table transformer

### PR DESCRIPTION
**Pull request by uarlouski**
Closes #3137

### CODE REVIEW

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by ikalinin1 — APPROVED** [review]

File: docs/modules/plugins/pages/plugin-rest-api.adoc [hunk 1]
```diff
@@ -191,6 +191,82 @@ When I send HTTP GET to the relative URL '/get?name=Content'
 Then `${response-code}` is equal to `200`
 ----
 
+= Table Transformers
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-rest-api.adoc:194]: here should be a link to glossary

File: docs/modules/plugins/pages/plugin-rest-api.adoc [hunk 2]
```diff
@@ -191,6 +191,82 @@ When I send HTTP GET to the relative URL '/get?name=Content'
...
+|`url`
+|The URL of the JSON resource that can be retrieved using the https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/GET[HTTP GET] call.
+
+|columns
+a|The semicolon-separated list of mappings between column names and {json-paths} used to get column values.
+
+.Columns parameter format
+[source]
+----
+columns=column_name=json_path;column_name=json_path
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-rest-api.adoc:219]:
```suggestion
columns=column_name_1=json_path_1;column_name_2=json_path_2
```

File: docs/modules/plugins/pages/plugin-rest-api.adoc [hunk 3]
```diff
@@ -191,6 +191,82 @@ When I send HTTP GET to the relative URL '/get?name=Content'
...
+
+.Usage of FROM_JSON transformer
+[source,gherkin]
+----
+Given request body: {
+  "fullName": "<fname> <lname>"
+}
+When I execute HTTP POST request for resource with URL `https://service-under-test.com/register`
+Examples:
+{transformer=FROM_JSON, url=https://fakedata.io/users, columns=fname=$..firstName;lname=$..lastName}
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-rest-api.adoc:267]:
```suggestion
{transformer=FROM_JSON, url=https://data-provider/users, columns=fname=$..firstName;lname=$..lastName}
```

**Review by valfirst — CHANGES_REQUESTED** [review]

File: docs/modules/plugins/pages/plugin-rest-api.adoc (hunk 1 above)
Comment (valfirst) [reply docs/modules/plugins/pages/plugin-rest-api.adoc:194]:
```suggestion
== xref:ROOT:glossary.adoc#_table_transformer[Table Transformers]
```

**Review by valfirst — APPROVED** [review]

**Pull request merged**